src/main/kotlin/de/bund/digitalservice/useid/tracking/WidgetTracking.kt

```diff
@@ -0,0 +1,29 @@
+package de.bund.digitalservice.useid.tracking
+
+import org.springframework.stereotype.Component
+
+@Component
+class WidgetTracking {
+
+    class Categories {
+        val widget = "Widget"
+    }
+
+    class Actions {
+        val loaded = "loaded"
+        val buttonPressed = "buttonPressed"
+        val error = "error"
+    }
+
+    class Names {
+        val widget = "widget"
+        val startIdent = "startIdent"
+        val incompatible = "incompatible"
+        val unsupported_os = "unsupported_os"
```

Comment:
camelCase and snake_case mixed?
If wished like that in Matomo, I'd say we should at least only use one in our code.

Reply:
I agree